bin/json_schema.py

```diff
@@ -182,6 +191,22 @@ def _handle_parameter_selection(self, d: dict, param_index: int) -> dict:
                     tmp_param_dict[param_name] = param_value[param_index]
                 return {"name": key, "params": tmp_param_dict}  
                 
+    
+    def unique_dicts_in_list(self, dict_list: list) -> list:
+        """
+        function is pretty straight forwrd: it checks if all elements in a list are unique and returns only unique once.
```

Comment:
maybe you can add comment to explain that you want to get the unique dictionaries from a list of dictionaries.